config/crd/bases/_.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
(remove file from commit, this is an issue in the code base that needs to be resolved)